### PR TITLE
Add payment review box component

### PR DESCRIPTION
**Pull request by aftabrehan**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -11,6 +12,7 @@ const App = () => {
       </header> */}
       <main className="main">
         <Outlet />
+        <Payment />
```
Comment (mrxshinji) [inline src/App.jsx:15]: to move it into routes and separate the item.

Comment (NadiaFrShLm) [reply src/App.jsx]: done, payment component is in the routes folder

**Review by mrxshinji — CHANGES_REQUESTED** [review]
Hi Nadia, 
Good work so far, just 2 more things to do 

@ index.js 
add under children
```
      {
        path: 'stories/payment',
        element: <Payment />,
      },
```

@story-page.js 
add 
```
<Link to={`/stories/payment`}> Payment </Link>
```

this will allow us to navigate to view the component made

**Comment by mrxshinji** [comment]
Fixes #10

**Pull request merged**